superset-frontend/src/dashboard/components/PropertiesModal/PropertiesModal.test.jsx

```diff
@@ -53,26 +58,30 @@ fetchMock.get('glob:*/api/v1/dashboard/*', {
   },
 });
 
+const requiredProps = {
+  dashboardId: 1,
+  show: true,
+  addSuccessToast: () => {},
+};
+
+const setup = overrideProps =>
+  mount(
+    <Provider store={mockStore}>
+      <PropertiesModal {...requiredProps} {...overrideProps} />
+    </Provider>,
+    {
```

Comment:
Doesn't seem like a blocker, but instead of adding the wrappingComponent and wrappingComponentProps, I think you can use `import { styledMount as mount } from 'spec/helpers/theming';` at the top and get all that for free.